apps/explorer/src/api/assets/assetsHooks.ts

```diff
@@ -169,3 +169,28 @@ export const useAssetsMetadata = (assetIds: string[] = []) => {
     return returnedVerifiedTokensMetadata
   } else return returnedCompleteMetadata
 }
+
+// TODO: get list from backend (enum?)
+// See: https://github.com/alephium/explorer-backend/issues/512
+export const useTokensWithAvailablePrice = () => ['ALPH', 'USDT', 'USDC', 'DAI', 'WBTC', 'WETH', 'AYIN']
```

Comment:
Why not do the same as in the wallets where we query every verified token? If we get a price, good. If we don't, also good. We keep track of what we queried so we don't query again.

Reply:
Just to reduce the number of useless queries

Reply:
ok, explorer is easily update-able in contrary to wallets that require user to upgrade. Approving.